samples/crypto/aes_gcm/README.rst

```diff
@@ -7,22 +7,27 @@ Crypto: AES GCM
    :local:
    :depth: 2
 
+The AES GCM sample shows how to perform AES authenticated encryption and decryption operations using the GCM block cipher mode and a 128-bit AES key.
```

Comment:
Duplication of "mode" since GCM is an acronym for "Galois/Counter Mode". I would prefer algorithm to stay consistent with CCM

Reply:
GCM algorithm getting added?

Reply:
Changing to:
The AES GCM sample shows how to perform AES authenticated encryption and decryption operations using the GCM algorithm and a 128-bit AES key.